SPEC.md

```diff
@@ -697,30 +694,55 @@ contract implementation.
 #### 4.7.3. Public Randomness Storage
 
 **PUB_RAND_VALUES**: Individual public randomness values
-- Type: `Map<(&[u8], u64), Vec<u8>>`
+- Type: `Map<(u64, &[u8]), Vec<u8>>`
 - Storage key: `"pub_rand_values"`
-- Key format: `(fp_pubkey_bytes, block_height)`
-- Purpose: Stores individual public randomness values revealed during finality
-  signature submission
-
-**PUB_RAND_COMMITS**: Public randomness commitments
-- Type: `Map<(&[u8], u64), PubRandCommit>`
-- Storage key: `"pub_rand_commits"`
-- Key format: `(fp_pubkey_bytes, start_height)`
-- Structure:
-  ```rust
-  pub struct PubRandCommit {
-    /// The height of the first commitment
-    pub start_height: u64,
-    /// The amount of committed public randomness
-    pub num_pub_rand: u64,
-    /// The epoch number of Babylon when the commit was submitted
-    pub babylon_epoch: u64,
-    /// Value of the commitment.
-    /// Currently, it's the root of the Merkle tree constructed by the public randomness
-    pub commitment: Vec<u8>,
```

Comment:
Why is this part gone? I guess AI did this 😅

Reply:
I think it's because the key format changed. Will check.